modules/highgui/src/window.cpp

```diff
@@ -1243,8 +1241,8 @@ int cv::createButton(const String&, ButtonCallback, void*, int , bool )
 #define CV_NO_GUI_ERROR(funcname) \
     cv::error(cv::Error::StsError, \
     "The function is not implemented. " \
-    "Rebuild the library with Windows, GTK+ 2.x or Cocoa support. "\
-    "If you are on Ubuntu or Debian, install libgtk2.0-dev and pkg-config, then re-run cmake or configure script", \
+    "Rebuild the library with Windows, Cocoa, Qt or Plugin support. "\
+    "If you are on Ubuntu or Debian, install qtbase5-dev, then re-run cmake", \
```

Comment:
> qtbase5-dev

Active Qt support is abandoned by OpenCV core dev team due to Qt licensing issues.
We still recommend GTK3/2 backend by default on Linux.

Reply:
> due to Qt licensing issues.

are there some details somewhere? As far as as I can see, it is LGPL3 + lots of FUD to sell their commercial stuff. GTK is LGPL2.

Also, `pip install opencv-python` gives you Qt for UI and I think it would be nice to be consistent here.